aspnetcore/fundamentals/error-handling.md

```diff
@@ -34,7 +34,8 @@ The page includes the following information about the exception and the request:
 * Cookies (if any)
 * Headers
 
-To see the Developer Exception Page in the sample app, use the `DevEnvironment` preprocessor directive and select **Trigger an exception** on the home page.
+To see the Developer Exception Page in the 
+[sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/error-handling/samples/2.x), use the `DevEnvironment` preprocessor directive and select **Trigger an exception** on the home page.
```

Comment:
fyi ... I always link to the *samples* folder and let the reader pick the version. No need to update the links that way going forward.

Reply:
That's what I do too.